packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -331,11 +331,17 @@ export class DocumentDeltaConnection
 	/**
 	 * Submits a new signal to the server
 	 *
-	 * @param message - signal to submit
+	 * @param content - Content of the signal.
+	 * @param targetClientId - When specified, the signal is only sent to the provided client id.
 	 */
-	public submitSignal(message: IDocumentMessage): void {
+	public submitSignal(content: IDocumentMessage, targetClientId?: string): void {
 		this.checkNotDisposed();
-		this.emitMessages("submitSignal", [[message]]);
+
+		if (targetClientId && this.details.supportedFeatures?.submit_signals_v2 !== true) {
+			throw new UsageError("Sending signals to specific client ids is not supported.");
+		}
```

Comment:
@GaryWilber, I am trying to understand this bit of logic.
1. Even if supportedFeatures.submit_signals_v2 was true, nothing happens here with targetClientId.
2. odsp-driver overrides this method. It has no such check. Is there some case where the override is not used?
cc: @WillieHabi